poolmanager/server/controlplane/controlplane.go

```diff
@@ -0,0 +1,20 @@
+/*
+	Interface between the Node Pool Manager and the Control Plane
+*/
+
+package controlplane
+
+const CAPACITY_PER_RUNNER = 4096
```

Comment:
Go's idiom for constants is mixed casing for constants https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps (and they note this is odd, but the stdlib is this way)